.travis.yml

```diff
@@ -1,10 +1,13 @@
 # Use Ubuntu 14.04
 dist: trusty
+# Can't use sudo: false because of travis-ci/travis-ci#7019
+sudo: required
```

Comment:
@ashawley Wanna try bumping back up to jdk 8? Looks we're using jdk 7 due to a stackoverflow in 2014: https://github.com/sbt/website/pull/4#issuecomment-44973507.